hack/vendor.sh

```
@@ -6,14 +6,20 @@ vendor() {
   go mod verify
 }
 
-echo "Updating dependencies for Cluster CAPI Operator"
-vendor
+vendor_in_subdir() {
+  local dir=$1
+  echo "Updating dependencies for $dir"
+  cd "$dir" && GOWORK=off vendor && cd -
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)